### DISCUSSION TITLE
fly.toml not using `source = litefs`

### DISCUSSION THREAD

**Discussion by nicolas-besnard** [epicweb-dev/epic-stack#830] [Q&A]
Hi,

By looking at [Fly's documentation](https://fly.io/docs/litefs/getting-started-fly/#file-system) about LiteFS, I realised it says that the mount section should be as follow:

```yml
[mounts]
  source = "litefs"
  destination = "/var/lib/litefs"
```

But in this project, we have

```yml
[mounts]
source = "data"
destination = "/data"
```

Is there any reason why this project is not using a `lifets` source?

**Comment by kentcdodds** [comment]
It doesn't really make a difference when it's called. I'm pretty sure I put my solution together before those docs existed. Additionally, the volume that we are mounting isn't exclusively for LiteFS necessarily.
